artifacts/definitions/Windows/Applications/SBECmd.yaml

```diff
@@ -0,0 +1,96 @@
+name: Windows.Applications.SBECmd
+description: |
+    Execute Eric Zimmerman's SBECmd and return output for analysis
+    
+    Objective:
+    
+    - Find which folders were accessed on the local machine, the network, and/or removable devices. Evidence of previously existing folders after deletion/overwrite. When certain folders were accessed.
+
+    Interpretation:
+    
+    - Stores information about which folders were most recently browsed by the user.
+    
+    MITRE ATT&CK ID: TA0009 - Collection
+    
+author: Eduardo Mattos - @eduardfir
+
+reference:
+  - https://github.com/EricZimmerman
+  
+required_permissions:
+  - EXECVE
+
+type: CLIENT
+
+tools:
+  - name: SBECmd
+
+precondition: SELECT OS From info() where OS = 'windows'
+
+parameters:
+  - name: tactic
```

Comment:
These were from an idea from our friend MattG, to support a future feature of hunting in accordance to MITRE TTPS. Example: IR case where client was hit with Sodin. We know what Sodin's TTPS are and we can create a hunt based on them (e.g.: https://thedfirreport.com/2021/03/29/sodinokibi-aka-revil-ransomware/) 
They are not useful at this moment however, it was more of a preparation for the future, so it could be removed